### ISSUE TITLE
Thought a gigabyte of free space would be enough to make a backup. It wasn't. 

### ISSUE THREAD

**Issue by ImNotRice** [L1ghtmann/IAmLazy#14]
Thought a gigabyte of free space would be enough to make a backup. It wasn't. Now my phone storage is completely filled up (I acknowledge this is my fault). Is there a way I can manually delete the backups through Filza or something else? The Delete A Backup feature doesn't work because while backing up it failed in the middle of the process so it isn't recognized as one.

Edit an hour later: Reddit is back up, I looked through some threads and Light said it should save to 
/var/mobile/Documents/me.lightmann.iamlazy/
I looked there with Filza, but there’s no “me.lightmann.iamlazy” folder or file. However, I did find something in 
/private/var/tmp/me.lightmann.iamlazy
There were hundreds of folders with subfolders. could this be what i have to delete? I assume the backup automatically transfers from the temp folder to the Documents folder when it’s done but since it failed for me, it stayed in the temp folder.

**Comment by UsrLightmann** [comment]
Hey!

Thanks for reporting this. 

Yes, it's safe to delete the contents of /var/tmp/me.lightmann.iamlazy/ and it serves exactly the purpose you suspected -- acting as a "middleman" during the backup process.

Do you happen to have a handful of large packages installed? Themes, applications, etc?

**Comment by ImNotRice** [comment]
Hey Light! Thanks for the quick response. I have around 90 tweaks, 10 of them being applications and 2 of them being themes. Not sure how to check the amount of space they take up, unfortunately.

**Comment by UsrLightmann** [comment]
You can check by running the following in a terminal (NewTerm, MTerm, etc): 
```
dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | grep -v -e gsc. -e cy+ -e firmware 

```
This will display your installed packages in ascending order according to their _estimated_ size in kb.

**Comment by davidiesel** [comment]
I am having problems with I am lazy amd I am speedy does nothing but spins around

**Comment by UsrLightmann** [comment]
@davidiesel I messaged you in the IAmSpeed issue you made and sent you an email a few weeks back. Please don't post on irrelevant issues.

**Comment by ImNotRice** [comment]
Darkpapers, appstore++, iCleaner Pro, iSecureOS, XenHTML, and altdaemon are the top of the list

**Comment by davidiesel** [comment]
> @davidiesel I messaged you in the IAmSpeed issue you made and sent you an email a few weeks back. Please don't post on irrelevant issues. 

- [ ] Bro nothing like that i am still facing issues of both i am speed and i am lazy which i love and really don't know why it just goes spinning on the 1st part and does nothing after that. I even rootfs then it worked fine amd after that again the same issue tried renaming the folders as old but stil nothing thats all bro

**Comment by davidiesel** [comment]
@UsrLightmann bro i sent you an email regarding both. Havent received your reply?

**Comment by UsrLightmann** [comment]
I've attempted to fix this issue in v1.4.0 which should be live in the next 12 hrs. If you could give it a go once it's been released and let me know if your issue has indeed been resolved, that would be super helpful. Thanks in advance!

**Comment by ImNotRice** [comment]
Hey! Unfortunately, I recently bought a new iPhone which was running iOS 15 straight out of the box. Unfortunately, I won’t be able to test it anymore. Sorry about that.

**Comment by davidiesel** [comment]
I will dont worry just let me know when 1.40 is out for both

David Oliveira 
JMJ


On 06-Jan-2022, at 10:54, ImNotRice ***@***.***> wrote:

﻿

Hey! Unfortunately, I recently bought a new iPhone which was running iOS 15 straight out of the box. Unfortunately, I won’t be able to test it anymore. Sorry about that.

**Comment by davidiesel** [comment]
Sure bro i will love your wirk👊🏽

David Oliveira 
JMJ


On 06-Jan-2022, at 11:11, David Oliveira ***@***.***> wrote:

﻿ I will dont worry just let me know when 1.40 is out for both

David Oliveira 
JMJ


On 06-Jan-2022, at 10:54, ImNotRice ***@***.***> wrote:

﻿

Hey! Unfortunately, I recently bought a new iPhone which was running iOS 15 straight out of the box. Unfortunately, I won’t be able to test it anymore. Sorry about that.

**Comment by L1ghtmann** [comment]
Closing since this should be fixed as of v1.4.0.
